Gems/AtomLyIntegration/CommonFeatures/Code/Source/PostProcess/ChromaticAberration/EditorChromaticAberrationComponent.cpp

```diff
@@ -0,0 +1,101 @@
+        {
+            BaseClass::Reflect(context);
+
+            if (AZ::SerializeContext* serializeContext = azrtti_cast<AZ::SerializeContext*>(context))
+            {
+                serializeContext->Class<EditorChromaticAberrationComponent, BaseClass>()->Version(1);
+
+                if (AZ::EditContext* editContext = serializeContext->GetEditContext())
+                {
+                    editContext->Class<EditorChromaticAberrationComponent>("Chromatic Aberration", "Controls the Chromatic Aberration")
+                        ->ClassElement(Edit::ClassElements::EditorData, "")
+                        ->Attribute(Edit::Attributes::Category, "Atom")
+                        ->Attribute(
+                            AZ::Edit::Attributes::Icon, "Icons/Components/Component_Placeholder.svg") // [GFX TODO ATOM-2672][PostFX] need to create icons for PostProcessing.
+                        ->Attribute(
+                            AZ::Edit::Attributes::ViewportIcon,
+                            "Icons/Components/Viewport/Component_Placeholder.svg") // [GFX TODO ATOM-2672][PostFX] need to create icons for PostProcessing.
+                        ->Attribute(Edit::Attributes::AppearsInAddComponentMenu, AZ_CRC("Game", 0x232b318c))
+                        ->Attribute(Edit::Attributes::AutoExpand, true)
+                        ->Attribute(
+                            Edit::Attributes::HelpPageURL,
+                            "https://o3de.org/docs/user-guide/components/reference/atom/ChromaticAberration/") // [TODO ATOM-2672][PostFX] need create page for PostProcessing.
+                        ;
+
+                    editContext->Class<ChromaticAberrationComponentController>("ChromaticAberrationComponentController", "")
+                        ->ClassElement(AZ::Edit::ClassElements::EditorData, "")
+                        ->Attribute(AZ::Edit::Attributes::AutoExpand, true)
+                        ->DataElement(AZ::Edit::UIHandlers::Default, &ChromaticAberrationComponentController::m_configuration, "Configuration", "")
+                        ->Attribute(AZ::Edit::Attributes::Visibility, AZ::Edit::PropertyVisibility::ShowChildrenOnly);
+
+                    editContext->Class<ChromaticAberrationComponentConfig>("ChromaticAberrationComponentConfig", "")
+                        ->DataElement(
+                            Edit::UIHandlers::CheckBox, &ChromaticAberrationComponentConfig::m_enabled, "Enable Chromatic Aberration", "Enable Chromatic Aberration.")
+                        ->Attribute(Edit::Attributes::ChangeNotify, Edit::PropertyRefreshLevels::ValuesOnly)
+
+                        ->DataElement(AZ::Edit::UIHandlers::Slider, &ChromaticAberrationComponentConfig::m_strength, "Strength", "Strength of effect")
+                        ->Attribute(AZ::Edit::Attributes::Min, 0.0f)
+                        ->Attribute(AZ::Edit::Attributes::Max, 1.0f)
```

Comment:
It does indeed and the higher strengths aren't useful. I didn't want to include an unnecessary divide though. Do you think the max should be limited to 0.1 or something? Could that confuse a user?

Reply:
These kind of factors are always tricky to setup, a value in the [0,1] would be better, but a division is worse. We should at least put the slider range on some values we think are expected, to avoid loosing too much precision on the slider.